### PR TITLE
chore(elements): update mdc packages to v13

### LINKED ISSUE #341: Update material-components-web to v10.0.0
### Feature request
We should update all material web components to the latest version 10.0.0 to benefit from the latest features and bug fixes.

### LINKED ISSUE #456: Update material-components-web to the latest version
We should update the mdc package to the latest version.

### PR DESCRIPTION
**Pull request by P1NHE4D**
✨ **Features:**
- Upgrade all `@material` packages to ~~v10.0.0~~ v13.0.0
- Update `@stencil/sass` to support new sass functionalities of the `@material` package 
- `<ino-chip>`
  - add `chipClicked` event
  - add `disabled` property

⚠️ **Breaking changes:**
- `ino-progress-bar`: removed `ino-reversed` property as it is no longer supported by the `mdc-linear-progress` component
- `ino-chip-set`: removed
- `ino-chip`:
  - remove deprecated `icon` property
  - rename event `removeChip` to `chipRemoved`
  - `chipRemoved` now emits `value` as its detail instead of the element itself (element can still be accessed by `event.target`)
- `ino-switch`
  - All CSS-Vars have been removed in order to keep a more consistent look across the library and also to reduce the maintenance cost
  - MDC provides utilities to make it more straightforward to customize the component. Sadly, this utility is not compatible with CSS variables

🎟️ **Closes #341**
🎟️ **Closes #456**

### CODE REVIEW

File: packages/elements/src/components/ino-range/readme.md [hunk 1]
```diff
@@ -80,19 +80,34 @@ class MyComponent extends Component {
...
+
+
+## Methods
+
+### `setFnToMapValueToAriaText(fn: (value: number) => string) => Promise<void>`
+
+Should be used to make the component accessible.
+If the value is not user-friendly (e.g. a number to represent the day of the week),
+use this method to set a function that maps the slider `value` to value of the `aria-valuetext` attribute (e.g. `0` => `monday`).
+
```
Comment (pfecht) [inline packages/elements/src/components/ino-range/readme.md:105]: May be good to add an example here for clarification:

```
range = document.querySelector("ino-range")
range.setFnToMapValueToAriaText((value: number) => `${value}. day in this week`)
```

File: packages/elements/src/components/ino-select/ino-select.tsx [hunk 2]
```diff
@@ -227,7 +227,7 @@ export class Select implements ComponentInterface {
             />
           </div>
           <div class="mdc-select__menu mdc-menu mdc-menu-surface mdc-menu-surface--fullwidth">
-            <ul class="mdc-list" ref={(el) => (this.mdcOptionsListEl = el)}>
+            <ul class="mdc-deprecated-list" ref={(el) => (this.mdcOptionsListEl = el)}>
```
Comment (pfecht) [inline packages/elements/src/components/ino-select/ino-select.tsx:230]: Do we need a follow up to move away from deprecated list?

File: packages/storybook/src/stories/ino-sidebar/ino-sidebar.stories.ts [hunk 3]
```diff
@@ -18,6 +17,8 @@ const eventHandler = (e) => {
     .closest('.sidebar-demo')
     .querySelector('ino-sidebar');
 
+  console.log(sidebar)
```
Comment (pfecht) [inline packages/storybook/src/stories/ino-sidebar/ino-sidebar.stories.ts:20]: remove this

File: packages/storybook/src/stories/ino-switch/ino-switch.stories.ts [hunk 4]
```diff
@@ -7,6 +7,7 @@ import './ino-switch.scss';
 
 const eventHandler = (e) => {
   const el = e.target;
+  console.log(e);
```
Comment (pfecht) [inline packages/storybook/src/stories/ino-switch/ino-switch.stories.ts:10]: remove this

**Review by pfecht — APPROVED** [review]

File: packages/elements/src/components/ino-select/ino-select.tsx (hunk 2 above)
Comment (janivo) [reply packages/elements/src/components/ino-select/ino-select.tsx:230]: https://github.com/inovex/elements/issues/541

**Review by janivo — APPROVED** [review]

**Pull request merged**